src/main/java/com/thedariusz/warnme/user/User.java

```diff
@@ -0,0 +1,78 @@
+package com.thedariusz.warnme.user;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import java.util.Set;
+
+@Entity
+public class User {
```

Comment:
You should use the common naming practice where each entity also ends with Entity suffix. ->`UserEntity`